api/jobs.go

```diff
@@ -97,6 +92,9 @@ func (api *API) CreateJobHandler(w http.ResponseWriter, req *http.Request) {
 		}
 	}
 
+	newJob.Links.Self = host + "/v1" + newJob.Links.Self
```

Comment:
**COMMENT 1**
The `/v1` should not be hard coded. Better to either use a constant or for the value to be passed in